api/python/tests/test_bucket.py

```diff
@@ -314,7 +314,11 @@ def test_search_bucket(self, config_mock, search_mock):
         b.search('blah', limit=1)
 
         config_mock.assert_called_once_with('navigator_url')
-        search_mock.assert_called_once_with('blah', 'https://es-fake.endpoint', limit=1, aws_region='us-meow')
+        search_mock.assert_called_once_with('blah',
+                                            'https://es-fake.endpoint',
+                                            limit=1,
+                                            aws_region='us-meow',
```

Comment:
maybe drop the supertroopers reference, this is a public repo.

Reply:
Ha! I didn't add that or even notice the Super Troopers connection. :-)